tidb-cloud/set-up-private-endpoint-connections.md

```diff
@@ -36,73 +36,79 @@ This section describes how to set up a private endpoint with AWS PrivateLink.
 
 > **Note:**
 >
 > The default region is where your cluster is located. Do not change it. Cross-region private endpoint is currently not supported.
 
-#### Step 4. Create an AWS interface endpoint
+### Step 3. Create an AWS interface endpoint
 
 At this stage, TiDB Cloud begins creating an endpoint service, which takes 3 to 4 minutes. During the creation process, perform the following operations:
 
 1. Fill in the **VPC ID** and **Subnet IDs** fields. You can get the IDs from your AWS Management Console.
 
-    If you do not know how to get these IDs, click **Show Instruction** and you will see two screenshots for reference.
-
-2. After the endpoint service is created, check the command in the **Create VPC Interface Endpoint** area, and take a note of your endpoint service name.
+2. After the endpoint service is created, take a note of your endpoint service name from the command.
 
     ![Endpoint service name](/media/tidb-cloud/private-endpoint/private-endpoint-service-name.png)
 
-3. Create a VPC interface endpoint in AWS. You can either use the AWS Management Console or the AWS CLI.
+3. Create a VPC interface endpoint in AWS.
+
+    You can either use the AWS Management Console or the AWS CLI.
 
     <SimpleTab>
     <div label="Use AWS Console">
 
     To use the AWS Management Console to create the VPC interface endpoint, perform the following steps:
 
-    1. In your AWS Management Console, go to **VPC** > **Endpoints**, and click **Create Endpoint** in the upper-right corner. The **Create endpoint** page is displayed.
+    1. Go to **VPC** > **Endpoints**.
+    2. Click **Create Endpoint**.
+
+        The **Create endpoint** page is displayed.
 
-        ![Create endpoint](/media/tidb-cloud/private-endpoint/create-endpoint-1.png)
+        <img src="../media/tidb-cloud/private-endpoint/create-endpoint-1.png" width="70%" auto="Create endpoint" />
 
-    2. Under **Service category**, select **Other endpoint services**.
-    3. Under **Service settings**, enter the endpoint service name you have obtained from the **Interface endpoint** page of the TiDB Cloud console, and click **Verify service**.
+    3. Select **Other endpoint services**.
+    4. Enter the endpoint service name you have obtained from the **Interface endpoint** page of the TiDB Cloud console.
+    5. Click **Verify service**.
 
-        ![Verify endpoint service](/media/tidb-cloud/private-endpoint/create-endpoint-2.png)
+        <img src="../media/tidb-cloud/private-endpoint/create-endpoint-2.png" width="70%" auto="Verify endpoint service" />
 
-    4. After the service name is verified, under **VPC**, select your VPC in the drop-down list. Then the pre-populated **Subnets** area is displayed.
-    5. In the **Subnets** area, select the availability zones where your TiDB cluster is located. Then click **Create endpoint** at the bottom of the page.
+    6. Select your VPC in the drop-down list.
+    7. In the **Security groups** area, select up your security group properly.
```

Comment:
set or select (but not select up)? Please confirm. Because I see both words are used in the troubleshooting section.
```suggestion
    7. In the **Security groups** area, select your security group properly.
```

Reply:
+1, put the key verb first.

Reply:
@benmaoer @zhuangliangji1992 OK, I'll put the verb first, but from the console, maybe "select" is better? You have also suggested using "select" in the Lark group.
![image](https://user-images.githubusercontent.com/41534398/188107228-b401e487-4e89-4783-bbcd-33fbacae06cc.png)